packages/core/contracts/optimistic-asserter/implementation/test/AssertingCallerTest.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: AGPL-3.0-only
```

Comment:
Did you add the files below? It seems like this diff might be the result of a merge issue of some sort...?

Reply:
I synced our branch with yours, so I think diffs are now resolved.
Let me know if anything else doesn't fit with your branch.